util/options.cc

```diff
@@ -691,6 +692,9 @@ font_options_t::get_font (void) const
   if (font_size_y == FONT_SIZE_UPEM)
     font_size_y = hb_face_get_upem (face);
 
+  hb_font_set_ppem (font, ppem, ppem);
```

Comment:
I didn't mean not having separate xppem and yppem. Just that both will be parsed from `--font-ppem`. Again, see what `--font-size` is doing for parsing.